src/styles/variables.scss

```diff
@@ -0,0 +1,31 @@
+//colors
```

Comment:
seems like this file should not be in that PR, please remove it

Reply:
removed this file.